src/System.Net.Security/tests/FunctionalTests/DummyTcpServer.cs

```diff
@@ -140,40 +143,25 @@ private void OnAccept(IAsyncResult result)
                 if (useSsl)
                 {
                     state = new ClientState(client, sslEncryptionPolicy);
-                    Logger.LogInformation("Server: starting SSL authentication.");
+                    _log.WriteLine("Server: starting SSL authentication.");
 
 
                     SslStream sslStream = null;
                     X509Certificate2 certificate;
+                    certificate = new X509Certificate2("DummyTcpServer.pfx");
```

Comment:
Rather than adding a binary file to git (which makes it unhappy in the long run) how about hex-encoding the contents into a CertData.cs?